### ISSUE TITLE
Questions about dataset invalidation

### ISSUE THREAD

**Issue by Junzizaiji** [yoyo-yun/MA-Bert#4]
Dear author, the dataset link provided in your code is no longer valid, please provide a new link, thank you very much

**Comment by drunkprogrammer** [comment]
Yes, I keep the dataset. The link is https://pan.baidu.com/s/1iewWK8ZKgxsQQg497lGvBQ?pwd=ca9r
Password：ca9r .
